prescient/simulator/master_options.py

```diff
@@ -945,40 +421,30 @@ def _construct_inner_options_parser():
                                          action='store_true',
                                          dest='output_ruc_dispatches',
                                          default=False)
-    guiOverride['--output-ruc-dispatches'] = {}
-    guiOverride['--output-ruc-dispatches']['bpa'] = False
 
     output_simulation_options.add_option('--output-sced-initial-conditions',
                                          help='Output sced initial conditions prior to each solve. Default is False.',
                                          action='store_true',
                                          dest='output_sced_initial_conditions',
                                          default=False)
-    guiOverride['--output-sced-initial-conditions'] = {}
-    guiOverride['--output-sced-initial-conditions']['bpa'] = False
 
     output_simulation_options.add_option('--output-sced-demands',
                                          help='Output sced demands prior to each solve. Default is False.',
                                          action='store_true',
                                          dest='output_sced_demands',
                                          default=False)
-    guiOverride['--output-sced-demands'] = {}
-    guiOverride['--output-sced-demands']['bpa'] = False
 
     output_simulation_options.add_option('--output-sced-solutions',
                                          help='Output sced solutions following each solve. Default is False.',
                                          action='store_true',
                                          dest='output_sced_solutions',
```

Comment:
Never used, consider removing it.

Reply:
This comment is about --output-sced-solutions.

Reply:
Removed.